eth_utils/logging.py

```diff
@@ -94,7 +99,9 @@ class who's name corresponds to the import path for the class.
 
     logger_class = logging.Logger
 
-    def __new__(mcls, name, bases, namespace):
+    def __new__(
+        mcls: Type[THasLoggerMeta], name: str, bases: Any, namespace: Dict[str, str]
```

Comment:
- `bases` should probably be `Sequence[Type[Any]]`
- `namespace` should probably be `Dict[str, Any]`